src/pagingscrollview/src/NIPagingScrollView.m

```diff
@@ -64,6 +66,7 @@ - (void)commonInit {
   _scrollView = [[UIScrollView alloc] initWithFrame:self.bounds];
   _scrollView.pagingEnabled = YES;
   _scrollView.scrollsToTop = NO;
+  _scrollView.clipsToBounds = NO;
```

Comment:
I just checked and I believe the default is actually clipsToBounds=NO, but I was trying to be explicit about the need for it.